.circleci/config.yml

```diff
@@ -39,3 +42,16 @@ jobs:
                 docker push kidsfirstdrc/kf-ui-fhir-data-dashboard:kidsfirst
                 docker push kidsfirstdrc/kf-ui-fhir-data-dashboard:$GIT_COMMIT
             fi
+
+workflows:
+  build_and_test:
+    jobs:
+      - cypress/install:
+          build: 'npm run build'
+      - cypress/run:
```

Comment:
I think this runs the tests using Electron headless. Should we use the chrome executor to run the cypress tests since our users probably use that browser more? 

See chrome section under https://circleci.com/orbs/registry/orb/cypress-io/cypress

Reply:
I never looked into the new orb thing in Circle, looks cool though!

Reply:
Oh good call